submissions/migrations/0004_remove_django_extensions.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0003_submission_status'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='uuid',
+            field=models.UUIDField(default=uuid.uuid4, db_index=True),
+        ),
+    ]
```

Comment:
What's the deployment implications of this migration? How big is the submissions_submission table on production?